README.md

```diff
@@ -114,8 +114,8 @@ convert them to work weeks and work days and then show in human readable format.
 calculateWork :: Hour  -- type synonym for 'Time HourUnit'
               -> (WorkWeek, WorkDay)
 calculateWork workHours =
-    let completeWeeks = workWeek $ floor $ toUnit @WorkWeekUnit workHours
-        completeDays  = workDay  $ floor $ toUnit @WorkDayUnit  workHours - toUnit completeWeeks
+    let completeWeeks = floorUnit $ toUnit @WorkWeekUnit workHours
```

Comment:
I think you can also delete `workDay` and `workWeel` smart constructors from README since they are not used anymore. It will give impression that adding new unit is even simpler :slightly_smiling_face: